pkg/api/types.go

```diff
@@ -232,7 +232,7 @@ type Config struct {
 
 	// BuildVolumes specifies a list of volumes to mount to container running the
 	// build.
-	BuildVolumes VolumeList
+	BuildVolumes []string
```

Comment:
we're deferring to docker.

if you're using s2i directly, there is no security.